remote-workstations/new-agent-vm/Install-PCoIPAgent.sh

```diff
@@ -20,9 +20,10 @@
 # THE SOFTWARE.
 #
 
-if [[ $# -eq 1 ]]
+if [[ $# -eq 1 ]] || [[ $# -eq 2 ]]
```

Comment:
This seems like a potential bug down the road... I think the script should probably check that `BINARY_LOCATION` is set correctly when it tries to use it on successive invocations.